src/main/java/world/bentobox/bentobox/api/addons/AddonClassLoader.java

```diff
@@ -125,7 +125,11 @@ public static AddonDescription asDescription(YamlConfiguration data) throws Inva
         if (softDepend != null) {
             builder.softDependencies(Arrays.asList(softDepend.split("\\s*,\\s*")));
         }
-        builder.icon(Objects.requireNonNull(Material.getMaterial(data.getString("icon", "PAPER").toUpperCase(Locale.ENGLISH))));
+        Material icon = Material.getMaterial(data.getString("icon", "PAPER").toUpperCase(Locale.ENGLISH));
+        if (icon == null) {
```

Comment:
This code prevents issues if the addon's icon in addon.yml contains a material that doesn't exist anymore. This can happen when Minecraft versions change.